src/test/java/teammates/test/cases/action/StudentFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -695,6 +695,95 @@ public void testExecuteAndPostProcess() throws Exception {
                 r.getDestinationWithParams());
```

Comment:
You shall also have test for `InstructorFeedbackEditSaveAction`

Reply:
It is not possible to have that because of #8646. Replaced by UI test instead.

Reply:
👍